cg/services/analysis_starter/configurator/file_creators/nextflow/sample_sheet/nallo.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from pathlib import Path
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants.subject import PlinkPhenotypeStatus, PlinkSex
+from cg.io.csv import write_csv
+from cg.store.models import Case, CaseSample, Sample
+from cg.store.store import Store
+
+LOG = logging.getLogger(__name__)
+
+HEADERS: list[str] = [
+    "project",
+    "sample",
+    "file",
+    "family_id",
+    "paternal_id",
+    "maternal_id",
+    "sex",
+    "phenotype",
+]
+
+
+class NalloSampleSheetCreator:
```

Comment:
should we add a docstring saying that this class should follow the `SampleSheetCreator` protocol defined in `cg/services/analysis_starter/configurator/file_creators/nextflow/sample_sheet/protocol.py`?